### PR TITLE
PEM - threshold_unit update as per PEM-4455

### PR DESCRIPTION
**Pull request by nidhibhammar**

### CODE REVIEW

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx [hunk 1]
```diff
@@ -41,16 +41,16 @@ Use fields on the `General` tab to specify general information about the templat
 -   Use the `Target type` drop-down listbox to select the type of object that will be the focus of the alert.
 -   Use the `Applies to server` drop-down listbox to specify the server type (EDB Postgres Advanced Server or PostgreSQL) to which the alert will be applied; you can specify a single server type, or `ALL`.
 -   Use the `History retention` field to specify the number of days that the result of the alert execution will be stored on the PEM server.
--   Use the `Threshold unit` field to specify the unit type of the threshold value.
+-   Use the `Threshold unit` field to specify the unit type of the threshold value. Any meaningful unit according to the output type of the underlying SQL statement is defined here. For example - `MB`, `#` (number), `Days`, `%`, `No of leader ID's` , `Null`, `No of worker errors`, `Hours`, `Minutes`, `State`, or anything meaningful.
```
Comment (edbshubham) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx:44]: Please add examples for various units that we have used in system alert templates in PEM. Below are some for your reference-
days -> A user expires in N days
# -> Function Count
MB -> Average table bloat in server
Hours -> Last Vaccum
Minutes-> Number of minutes lag of replica server from primary server
STATE -> Server Down
% -> CPU Utilization
'' -> Events lagging in one slony cluster
NULL -> Swap consumption
For NULL type create a note as well like it is only defined for some system alert templates and can not be used for User-defined alert templates

Include below types saying something like - we can have any other threshold units also according to our need for ex.
No of leader ID's-> PGD Group Raft Leader ID not matching
No of worker errors -> PGD worker error detected

Comment (nidhibhammar) [reply product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx]: Updated the content as per the comment

**Review by edbshubham — APPROVED** [review]
Looking Good

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/01_pem_alerting_dialog.mdx [hunk 2]
```diff
@@ -37,13 +37,14 @@ Use the fields on the `General` tab to provide information about the alert:
...
+      -   Select a greater-than sign (>) to indicate that the alert should be triggered when the system values are greater than the values entered in the `Threshold values` fields.
+      -   Select a less-than sign (&lt;) to indicate that the alert should be triggered when the system values are less than the values entered in the `Threshold values` fields.
+  -   Use the threshold fields to specify the values that PEM will compare to the system values to determine if an alert should be raised. Please note that you must specify values for all three thresholds (`Low`, `Medium`, and `High`):
+      -   Enter a value that will trigger a low-severity alert in the `Low` field.
+      -   Enter a value that will trigger a medium-severity alert in the `Medium` field.
+      -   Enter a value that will trigger a high-severity alert in the `High` field.
+  -   The `Unit` field takes the value as per the threshold unit defined in the selected alert template. You cannot add or edit the value in the `Unit` field while creating or editing the alert. The value can be added or edited only when a custom alert template is created or edited.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/01_pem_alerting_dialog.mdx:47]:
```suggestion
  -   The `Unit` field takes the value as per the threshold unit defined in the selected alert template. You cannot add or edit the value in the `Unit` field while creating or editing the alert. The value can only be added or edited in the custom alert template.
```

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx [hunk 3]
```diff
@@ -41,18 +41,35 @@ Use fields on the `General` tab to specify general information about the templat
 -   Use the `Target type` drop-down listbox to select the type of object that will be the focus of the alert.
 -   Use the `Applies to server` drop-down listbox to specify the server type (EDB Postgres Advanced Server or PostgreSQL) to which the alert will be applied; you can specify a single server type, or `ALL`.
 -   Use the `History retention` field to specify the number of days that the result of the alert execution will be stored on the PEM server.
--   Use the `Threshold unit` field to specify the unit type of the threshold value.
+-   Use the `Threshold unit` field to specify the unit type of the threshold value. Any meaningful unit according to the output type of the underlying SQL statement is defined here. For example following units are used in the system alert templates:
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx:44]:
```suggestion
-   Use the `Threshold unit` field to specify a unit type of the threshold value that corresponds to the output type of the underlying SQL statement. For example, the following units are used in the system alert templates:
```

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx [hunk 4]
```diff
@@ -41,18 +41,35 @@ Use fields on the `General` tab to specify general information about the templat
...
+  | `''`           | Events lagging in one slony cluster |
+  | `days`         | A user expires in N days            |
+  | `Hours`        | Last Vaccum                         |
+  | `Minutes`      | Number of minutes lag of replica server from primary server |
+  | `MB`           | Average table bloat in server       |
+  | `NULL`         | Swap consumption                    |
+  | `STATE`        | Server down                         |
+
+  !!! Note
+      - `NULL` type is defined only for few system alert templates and cannot be defined for the custom alert templates.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx:59]:
```suggestion
      - `NULL` type is only defined for a few system alert templates and cannot be defined for the custom alert templates.
```

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx [hunk 5]
```diff
@@ -41,18 +41,35 @@ Use fields on the `General` tab to specify general information about the templat
...
+      - You can also define any meaningful `Threshold unit` like `No of leader ID's` is used for `PGD Group Raft Leader ID not matching` and `No of worker error` is used for `PGD worker error detected.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx:60]:
```suggestion
      - You can also define any meaningful `Threshold unit` such as:
         - `No of leader ID's` for `PGD Group Raft Leader ID not matching` 
         - `No of worker error` for `PGD worker error detected.
```

**Review by edbshubham — APPROVED** [review]
Looks good

**Pull request merged**